Mathlib/Algebra/Associated/Basic.lean

```diff
@@ -31,20 +31,20 @@ and prove basic properties of this quotient.
 assert_not_exists OrderedCommMonoid
 assert_not_exists Multiset
 
-variable {α : Type*} {β : Type*} {γ : Type*} {δ : Type*}
+variable {M N : Type*}
 
 section Prime
 
-variable [CommMonoidWithZero α]
+variable {M : Type*} [CommMonoidWithZero M]
```

Comment:
Why redeclare M here?

Reply:
AFAIR, I changed this section to `M`, then decided to change it everywhere but forgot to undo this change.
```suggestion
variable [CommMonoidWithZero M]
```